go.mod

```diff
@@ -46,6 +46,13 @@ require (
 	github.com/uber/jaeger-lib v2.4.1+incompatible
 	github.com/yookoala/gofast v0.7.0
 	github.com/yuin/gopher-lua v1.1.0
+	go.opentelemetry.io/otel v1.17.0
```

Comment:
Is it okay to have v1.17.0 and v1.16.0 mixed here?

Reply:
Good catch, its fixed now. :)